frontend/Settings/Layout.qml

```diff
@@ -89,6 +89,25 @@ Item {
                         settings.locale = locale
                     }
                 }
+                Controls.FormLabel {
+                    text: qsTr("Theme")
+                }
+                ComboBox {
+                    id: themeControl
+                    textRole: "text"
+
+                    model: ListModel {
+                        id: themeOptions
+                        ListElement {
+                            text: qsTr("Dark")
+                            locale: "dark"
```

Comment:
These could very-well be fine but what's `locale` for? Is this the locale specific value to use for the element? 
Seems like a weird key name